document_url/view/document_url_view.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id='action_ir_attachment_add_url' model='ir.actions.act_window'>
+        <field name='name'>Add URL</field>
+        <field name='res_model'>ir.attachment.add_url</field>
+        <field name='type'>ir.actions.act_window</field>
+        <field name='target'>new</field>
+        <field name='view_mode'>form</field>
+    </record>
+    <data>
```

Comment:
Remove data tag and deindent the rest (you can do it on the "pre-commit auto-fixes" one for reducing the noise).